src/centos/scripts/start-marklogic.sh

```diff
@@ -147,6 +147,49 @@ function restart_check {
     error "Failed to restart $1" exit
 }
 
+################################################################
+# setting default protocol and CA cert for curl calls to bootstrap host
+################################################################
+ML_BOOTSTRAP_PROTOCOL="http"
+ML_CACERT_FILE="/run/secrets/${MARKLOGIC_JOIN_CACERT_FILE}"
+
+################################################################
+# Check if TLS is enabled on Bootstrap host
+################################################################
+function validate_tls_parameters {
+    local bootstrap_status=$1 host_protocol
+    host_protocol=$(get_host_protocol "${MARKLOGIC_BOOTSTRAP_HOST}")
+    if [[ "${bootstrap_status}" != "localhost" ]]; then
```

Comment:
There are too many nested if statements, which reduce the code readability.
To reduce the nested If. Please try 
if [[ "${bootstrap_status}" != "localhost" ]]; then
    return
fi

Reply:
Agreed but nested if(s) are added here to handle different scenarios and print error messages that are expected to display.

Reply:
updated if statement